go.mod

```diff
@@ -41,7 +41,7 @@ require (
 	github.com/rancher/dynamiclistener v0.4.0-rc2
 	github.com/rancher/lasso v0.0.0-20240123150939-7055397d6dfa
 	github.com/rancher/rancher/pkg/apis v0.0.0-20240618132447-2145c8e9d603
```

Comment:
Would you mind updating this commit to a more recent value as well?

Reply:
Updated